app/pages/admin/products/edit/product-edit.service.js

```diff
@@ -0,0 +1,37 @@
+class ProductEditService {
+    constructor(firebase, toastService) {
+        'ngInject';
+
+        this.firebase = firebase;
+        this.toastService = toastService;
+    }
+
+    updateProduct(inputs, productId) {
+        if(inputs.title && inputs.category && inputs.price && inputs.imageUrl && inputs.amount && inputs.description) {
+            const updateProductData = {
+                'title': inputs.title,
```

Comment:
You don't need to put key as a string in JS object (if it's not with special chars)